Tests/UnitTests/APIToRequestMappingTests.swift

```diff
@@ -191,4 +191,12 @@ class APIToRequestMappingTests: XCTestCase {
         }
         XCTAssert(try session.size(window: "myWindow") == (width: 500, height: 500))
     }
+
+    func testSessionTouchScroll() throws {
+        let mockWebDriver: MockWebDriver = MockWebDriver()
+        let session = Session(webDriver: mockWebDriver, existingId: "mySession")
+        let element = Element(session: session, id: "myElement")
+        mockWebDriver.expect(path: "session/mySession/touch/scroll", method: .post)
+        try session.touchScroll(element: element, xOffset: 9, yOffset: 16)
+    }
```

Comment:
This definitely is good to keep, but I was thinking that we could just do a round-trip serialising and deserialising the type.  Perhaps a follow up can add those to ensure that we have a simple way to ensure that we have all the fields serialised and deserialised.

Reply:
We already support asserting on the serialized data in a closure, see the example of `testSessionFindElement`. Adding this would be nice.